docs/marina/api.md

```diff
@@ -8,17 +8,16 @@ image: /img/marina_logo.svg
 We recommend that all web developers read from the [Getting started](getting-started) section and onward .
 :::
 
-Marina injects a global API into websites visited by its users at window.marina. This API allows websites to request users' Liquid addresses and blinding keys, read data about the blockchain the user is connected to, and suggest that the user sign messages and send transactions.
+Marina injects a global API into websites visited by its users at window.marina. This API allows websites to request users' Liquid addresses and blinding keys, read data about the blockchain the user is connected to, and suggest that the user sign messages and send transactions. 
+
+The `marina-provider` package provides a function `detectProvider` to inspect and fetch the `window.marina` provider.
+
 
 ```javascript
-const marina = window.marina;
-if (marina) {
-  // From now on, this should always be true:
-  // marina === window.marina
-  initApp(marina); // initialize your app
-} else {
-  console.log("Please install Marina extension!");
-}
+import { detectProvider } from 'marina-provider';
```

Comment:
I would keep the branching. If it catches an error, even if we do not check what it its, a dev can assume marina is not installed if undefined